src/Runtime/OMResize.inc

```diff
@@ -0,0 +1,353 @@
+#ifdef __cplusplus
+#include <cassert>
+#else
+#include <assert.h>
+#endif
+
+#include <math.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "onnx-mlir/Runtime/OMTensor.h"
+
+/**
+ * The runtime implementation of ONNXResizeOp is manually translated python
+ * implementation: onnx/onnx/backend/test/case/node/resize.py (onnx v1.9.0)
+ * This implementation can be improved in efficiency.
+ **/
+
+static void linear_coeffs(float ratio, float *coeffs_buffer, int mode) {
+  coeffs_buffer[0] = 1 - ratio;
```

Comment:
I'm just looking to this code without context and wondering how can we guarantee that `coeffs_buffer` has at least 2 elements. Also `coeffs_buffer` could be null.   You could change the function signature to:
```
static void linear_coeffs(float ratio, float coeffs_buffer[2], int mode) {

Reply:
Also `mode` is not used, can you remove it pls

Reply:
The implementation is translated from python code. Some types and parameters could be optimized as you suggested. I may keep some interface as its python counter part before the code is thoroughly tested.